Sources/ApodiniDeployBuildSupport/DeploymentProvider.swift

```diff
@@ -24,6 +24,21 @@ public struct DeploymentProviderID: RawRepresentable, Hashable, Equatable, Codab
     }
 }
 
+public struct DeploymentStorage: MemoryStorage {
+    public static var current = DeploymentStorage()
+    
+    private var object: Object?
+    public typealias Object = WebServiceStructure
```

Comment:
As a Swift newbie: Couldn't we omit the `Object` typealias completely and let Swift infer the type? Or did you do that on purpose?

Reply:
You're right. Will remove that!